lib/db/schema/patch-033-034.sql

```diff
@@ -0,0 +1,38 @@
+CREATE TABLE IF NOT EXISTS unblockCodes (
+  uid BINARY(16) NOT NULL,
+  code VARCHAR(32) NOT NULL,
+  createdAt BIGINT SIGNED NOT NULL,
+  UNIQUE INDEX unblockCodes_uid_code(uid, code)
+) ENGINE=InnoDB;
+
+CREATE PROCEDURE `createUnblockCode_0` (
+    IN inUid BINARY(16),
+    IN inCode VARCHAR(32),
+    IN inCreatedAt BIGINT SIGNED
+)
+BEGIN
+    INSERT INTO unblockCodes(
```

Comment:
So, thanks to the `unique index`, it should return an error. This means that the auth-server perhaps should try a few times, the same way it does with creating `sessionTokens`.